src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 4 failures in `PowerShell-CI-macos`
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Operators/PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```